org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractXtextTests.java

```diff
@@ -64,9 +63,9 @@
 
 /**
  * @author Sven Efftinge - Initial contribution and API
- * @since 2.3
+ * @since 2.35
```

Comment:
wonder if we still should mark it as beta

Reply:
@cdietrich I noted that it was beta in some occurrence. For sure, we've been using it a lot ;)
I have no opinion on that.